internal/driver/commands.go

```diff
@@ -336,25 +336,22 @@ func listHelp(c string, redirect bool) string {
 }
 
 // browsers returns a list of commands to attempt for web visualization.
-// Commands which definitely will open a browser are prioritized over other
-// commands like xdg-open,  which may not open the javascript embedded SVG
-// files produced by the -web command in a browser.
 func browsers() []string {
 	var cmds []string
 	if userBrowser := os.Getenv("BROWSER"); userBrowser != "" {
 		cmds = append(cmds, userBrowser)
 	}
-	cmds = append(cmds, []string{"chrome", "google-chrome", "chromium", "firefox"}...)
 	switch runtime.GOOS {
 	case "darwin":
 		cmds = append(cmds, "/usr/bin/open")
 	case "windows":
 		cmds = append(cmds, "cmd /c start")
 	default:
+		cmds = append(cmds, []string{"chrome", "google-chrome", "chromium", "firefox", "sensible-browser"}...)
```

Comment:
Could you keep your comment about why the browser list is prioritized over xdg-open?

Reply:
Done.